docs/src/appendix/forge-library/deploy.md

```diff
@@ -1,31 +1,34 @@
 # `deploy`
 
-> `fn deploy(prepared_contract: PreparedContract) -> Result::<ContractAddress, RevertedTransaction>`
+```rust 
+trait ContractClassTrait {
+    fn deploy(self: @ContractClass, constructor_calldata: @Array::<felt252>) -> Result<ContractAddress, RevertedTransaction>;
+    ...
+}
+```
 
 Deploys a contract and returns its address.
 
-- `prepared_contract` - an instance of the struct `PreparedContract` that consists of the following fields:
+- `self` - an instance of the struct `ContractClass` that consists of the following fields:
   - `class_hash` - class hash of a previously [declared](declare.md) contract
-  - `constructor_calldata` - snapshot of calldata for the constructor
+
+- `constructor_calldata` - snapshot of calldata for the constructor
 
 ```rust
 use array::ArrayTrait;
 use result::ResultTrait;
-use cheatcodes::PreparedContract;
+use cheatcodes::{ declare, ContractClassTrait };
```

Comment:
Does it work without importing `ContractClassTraitImpl`?

Reply:
Nvm, it seems like it does in similar case for me

Reply:
Yep @Arcticae suggested removing it from exports and it works